modules/machine-health-checks-pausing.adoc

```diff
@@ -8,7 +8,12 @@
 [id="machine-health-checks-pausing_{context}"]
 = Pausing a MachineHealthCheck resource
 
-During the update process, nodes in the cluster might become temporarily unavailable. In the case of worker nodes, the machine health check might identify such nodes as unhealthy and reboot them. To avoid rebooting such nodes, pause all the `MachineHealthCheck` resources before updating the cluster.
+During the update process, nodes in the cluster might become temporarily unavailable. In the case of worker nodes, the MachineHealthCheck resources might identify such nodes as unhealthy and reboot them. To avoid rebooting such nodes, pause all the `MachineHealthCheck` resources before updating the cluster.
+
+[NOTE]
+====
+Some MachineHealthChecks might not need to be paused. If your MachineHealthCheck (MHC) resource relies on unrecoverable conditions, pausing that MHC is unnecessary.
```

Comment:
I'd probably just remove (MHC) here. You use `MachineHealthCheck` 3 times before introducing the abbreviation so it's probably not needed

```suggestion
Some `MachineHealthCheck` resources might not need to be paused. If your `MachineHealthCheck` resource relies on unrecoverable conditions, pausing that MHC is unnecessary.
```